templates/4_Post_Clustering.ipynb

```diff
@@ -48,7 +48,7 @@
   },
```

Comment:
<p>We can clear this stuff</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/832/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>